client/ayon_aftereffects/api/launch_logic.py

```diff
@@ -47,16 +47,15 @@ def main(*subprocess_args):
     launcher = ProcessLauncher(subprocess_args)
     launcher.start()
 
-    if os.environ.get("HEADLESS_PUBLISH"):
+    if os.environ.get("AYON_IN_TESTS"):
```

Comment:
I think we should use `is_in_tests` from `ayon_core.lib` so we don't have to tackle the env variable at all places.

Reply:
Replaced.